MetaMask/metamask-extension
# Revert "Update main-quote-summary designs/styles (#9612)"

This reverts commit 5456d55c88cb8934efdae936a78522a9064e5854 (#9612)

Explanation:  

This is being temporarily reverted because it has obscured whether the current quote is the best quote. This is a partial implementation of a design that moved the "Best Quote" label to the bottom section of the screen, which was outside the scope of this PR.

Manual testing steps:  
  - Click the "Swap" button on the home page
  - Select something for the source and destination token
  - Wait for quotes to be received
  - See that on the view quote screen, the current quote is marked as the "best quote", if there is a best quote.
